src/Pages/falcon9/stats/Stats.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import Stat from "./Stat/Stat";
+import "./Stats.scss";
+
+class Stats extends Component {
+  render() {
+    return (
+      <section id="Stats">
+        <article className="statsContents">
+          {STATS_INFO.map((el) => {
+            const { statName, statValue, statDescription } = el;
```

Comment:
```suggestion
          {STATS_INFO.map(({ statName, statValue, statDescription }) => {
```
함수 파라미터에서 바로 분리할 수 있습니다

Reply:
꿀팁 감사합니다 ㅎㅎ